src/Nethermind/Nethermind.Consensus/Rewards/RewardCalculator.cs

```diff
@@ -11,17 +11,31 @@ namespace Nethermind.Consensus.Rewards
 {
     public class RewardCalculator : IRewardCalculator, IRewardCalculatorSource
     {
-        private readonly ISpecProvider _specProvider;
+        private readonly ISpecProvider? _specProvider;
+        private readonly UInt256? _blockReward;
 
         public RewardCalculator(ISpecProvider? specProvider)
         {
             _specProvider = specProvider ?? throw new ArgumentNullException(nameof(specProvider));
         }
 
+        public RewardCalculator(UInt256? blockReward)
```

Comment:
StaticRewardCalculator just sets the reward for blocks, does not calculate anything, just a converter from IDictionary<long, UInt256> to List<BlockRewardInfo>. I need RewardCalculator because it does calculate reward for uncles and the main reward. I'm passing static blockReward value into constructor because It is specified in the tests (state.reward flag) and IReleaseSpec's reward value is not  the thing that I need (as per tests).